heartbeat/docker-compose

```diff
@@ -123,16 +113,16 @@ exit $OCF_SUCCESS
 if [ -r "$OCF_RESKEY_binpath" -a -x "$OCF_RESKEY_binpath" ]; then
 	COMMAND="$OCF_RESKEY_binpath"
 else
-	COMMAND=$(which docker-compose 2>/dev/null)
+	COMMAND="$OCF_RESKEY_binpath_default"
 fi
 
 DIR="$OCF_RESKEY_dirpath"
+PRE="$(echo ${DIR##*/} | tr A-Z a-z | sed 's/[^a-z0-9]//g')"
```

Comment:
We will have to do an `if [ -z "$OCF_RESKEY_svcname" ]; then` here, and
else with `PRE="$OCF_RESKEY_svcname"` to avoid it breaking for backwards compatibility.
```

Reply:
Well, the svcname is actually a buggy setting and should be replaced by the PREfix version. If just for consideration of backward compatibility, I'd rather to reset it to be inherited from $PRE. Is it okay?

Reply:
In my example code it set PRE to the svcname value if it's specified.

Reply:
No, the svcname is not equal to the PRE value. I wrongly use the svcname at the earlier version since I misunderstood the naming convention of docker-compose service. I just simply thought the service prefix was the same as we specified in the environment. However it is not the case, I've found the service prefix is from the directory name by removing punctuation with only alphabets and numbers remained. Using the svcname may cause inaccurate status determination to the docker processes and fail the status function.

Reply:
OK. I guess we're fine the way it is then.